DTS-STN/Alpha-Site
# Add canada.ca link to FIP

# Description

[Add link to FIP](https://trello.com/c/bwIyN7xe)

Add a link to canada.ca to the FIP.

Just links to the language choice page of canada.ca, but could make it link to language specific home page of canada.ca if anyone thinks that's a better idea.

For accessibility, I think just keeping the alt text of the image is fine. I've seen a `<span class='sr-only'>describe the link</span>` in some examples, but I'd be worried that would result in a screen reader reading the same text twice.

## Acceptance Criteria

The FIP links to canada.ca

## Test Instructions

1. Visit the home page
2. Click on the Government of Canada logo
3. Notice that it is a link to canada.ca

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
